src/output.jl

```diff
@@ -43,12 +47,15 @@ end
 plots the habitat suitability of a landscape for a given timestep t
 """
 function image_suitability(SD::Simulation_Data, t::Int)
```

Comment:
Image suitability now broken. Did you add a `.output`?

```
type Output has no field suitability

    getproperty@Base.jl:37[inlined]
    image_suitability(::MetaRange.Simulation_Data, ::Int64)@output.jl:51
    top-level scope@[Local: 1](http://localhost:1234/edit?id=2f728bd4-733b-11ee-205e-6745835035ab#)[inlined]
    ```

Reply:
that should already be fixed - I used "suitability" instead of "habitat", that's what I get for not being consistent with my names 😄